doc/release-notes.md

```diff
@@ -28,207 +26,77 @@ likely require a reindex.
-        - Instead of `addnode`, the `connection_type` field returns `manual`.
-        - Instead of `whitelisted`, the `permissions` field indicates if the peer has special privileges.
-
-- **`getblockfrompeer` Parameter Renaming**
-    - The named argument `block_hash` has been renamed to `blockhash` to align with the rest of the codebase.
-    - **Breaking Change:** If using named parameters, make sure to update them accordingly.
-
-- **`coinjoin stop` Error Handling**
-    - `coinjoin stop` will now return an error if there is no CoinJoin mixing session to stop.
-
-- **`getcoinjoininfo` Adjustments**
-    - `getcoinjoininfo` will no longer report `keys_left` and will not incorrectly warn about keypool depletion with descriptor wallets.
-
-- **`creditOutputs` Format Change**
-    - `creditOutputs` entries in various RPCs that output transactions as JSON are now shown as objects instead of strings.
-
-- **`quorum dkgsimerror` Argument Update**
-    - `quorum dkgsimerror` will no longer accept a decimal value between 0 and 1 for the `rate` argument.
-    - It will now expect an integer between **0** to **100**.
-
-- **Deprecated `protx *_hpmn` RPC Endpoints**
-    - Deprecated `protx *_hpmn` RPC entry points have been dropped in favor of `protx *_evo`.
-    - **Removed Endpoints:**
-        - `protx register_fund_hpmn`
-        - `protx register_hpmn`
-        - `protx register_prepare_hpmn`
-        - `protx update_service_hpmn`
+Build Changes
+-------------
 
-- **`governance` Descriptor Wallet Support**
-    - The `governance votemany` and `governance votealias` RPC commands now support descriptor-based wallets.
+The macOS distribution is no longer packaged in a disk image (DMG) and
+is now packaged in a ZIP archive. The macOS distribution is once again notarized.
 
-- **`createwallet` Behavior for Descriptor Wallets**
-    - When creating descriptor wallets, `createwallet` now requires explicitly setting `load_on_startup`.
+BIP324 / v2 P2P Protocol
+------------------------
 
-## Command-line Options
+Version 2 of the Dash P2P protocol / BIP324, which enables encryption of the p2p protocol
+has been enabled by default in this version. This was initially introduced in Dash Core
+v22.0.0 as an experimental feature, and has been now enabled by default. This change is
+backwards compatible, and connections to peers which do not support the v2 protocol will
+revert to using the v1 protocol.
 
-### Changes in Existing Command-line Options
+Network Changes
+---------------
+System ports, or ports that are lower than 1024 are now considered to be "bad" ports.
+As a result, other peers will avoid connecting to nodes that are listening on these ports.
+This change is to prevent potential DDoS attacks on services that are running on these ports.
+A number of other ports commonly used for authenticated services are also considered "bad" ports.
+You can view [the list of bad ports here](doc/p2p-bad-ports.md).
```

Comment:
```suggestion
System ports, or ports that are lower than 1024 are now considered to be "bad" ports.
As a result, other peers will avoid connecting to nodes that are listening on these ports.
This change is to prevent potential DDoS attacks against services running on these ports.
A number of other ports commonly used for authenticated services are also considered "bad" ports.
You can view [the list of bad ports here](doc/p2p-bad-ports.md).
```